amundsen_application/static/js/components/PreferencesPage/index.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { PreferenceGroup } from './PreferenceGroup';
+
+// TODO: Use css-modules instead of 'import'
+import './styles.scss';
+import {
+  ALL_NOTIFICATIONS_SUBTITLE,
+  ALL_NOTIFICATIONS_TITLE,
+  MINIMUM_NOTIFICATIONS_SUBTITLE,
+  MINIMUM_NOTIFICATIONS_TITLE,
+  NOTIFICATION_PREFERENCES_TITLE
+} from './constants';
+
+interface PreferencesPageState {
+  selectedPreference: string;
+}
+
+export interface DispatchFromProps {
+}
+
+export type PreferencesPageProps = DispatchFromProps;
+
+export class PreferencesPage extends React.Component<PreferencesPageProps, PreferencesPageState> {
+  constructor(props) {
+    super(props);
+    this.changePreference = this.changePreference.bind(this);
+
+    this.state = {
+      selectedPreference: ALL_NOTIFICATIONS_TITLE,
```

Comment:
We're going to want to be working with some sort of fixed `id` here because the constants can change and are only relevant to what is getting displayed in the UI. It shouldn't be the value used for executing actual functionality because it can change any time. If we go this route we will also need some helper methods for getting the title and subtitle that takes the `id` and returns which constants to use